src/hooks/sound/useSoundCreateInputs.tsx

```diff
@@ -41,6 +42,7 @@ const useSoundCreateInputs = () => {
     const shouldSplit = recipients.length !== 1
 
     const allocations = getSoundSplitAllocations(splitArgs)
+    const splitWallet = await getSplitWallet(splitArgs)
```

Comment:
are you confident the logic is the same for `sound splits` to generate the deterministic address?